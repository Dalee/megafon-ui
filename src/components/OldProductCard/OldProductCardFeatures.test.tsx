import * as React from 'react';
import { shallow } from 'enzyme';
import OldProductCardFeatures from './OldProductCardFeatures';

const props = {
    showcaseParams: [{
        title: 'Безлимитный интернет',
        value: 'на мессенджеры',
        children: [{
            title: 'Безлимитный интернет',
            caption: 'на видео, соц-сети и мессенджеры',
            value: 'test',
        }],
    },
    {
        children: [{
            title: 'Youtube, Rutube, Vimeo',
        }],
    },
    {
        children: [{
            title: 'WhatsApp, Viber, Facebook Messanger, Snapchat, eMotion',
        }],
    },
    {
        children: [{
            title: 'Вконтакте, Одноклассники, Facebook, Instagram, Twitter',
        }],
    }],
};

describe('<OldProductCardFeatures />', () => {
    it('it renders OldProductCardFeatures', () => {
        const wrapper = shallow(<OldProductCardFeatures {...props} />);
        expect(wrapper).toMatchSnapshot();
    });
});
