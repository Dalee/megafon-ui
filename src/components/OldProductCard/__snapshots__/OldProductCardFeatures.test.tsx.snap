// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OldProductCardFeatures /> it renders OldProductCardFeatures 1`] = `
<div
  className="old-product-card-features"
>
  <OldProductCardFeaturesTop
    className="old-product-card-features__top"
    params={
      Array [
        Object {
          "caption": "на видео, соц-сети и мессенджеры",
          "title": "Безлимитный интернет",
          "value": "test",
        },
      ]
    }
    socialIcons={Array []}
  />
  <OldProductCardFeaturesBottom
    className="old-product-card-features__bottom"
    key="0"
    params={
      Array [
        Object {
          "title": "Youtube, Rutube, Vimeo",
        },
      ]
    }
  />
  <OldProductCardFeaturesBottom
    className="old-product-card-features__bottom"
    key="1"
    params={
      Array [
        Object {
          "title": "WhatsApp, Viber, Facebook Messanger, Snapchat, eMotion",
        },
      ]
    }
  />
  <OldProductCardFeaturesBottom
    className="old-product-card-features__bottom"
    key="2"
    params={
      Array [
        Object {
          "title": "Вконтакте, Одноклассники, Facebook, Instagram, Twitter",
        },
      ]
    }
  />
</div>
`;
